src/services/results-calculator.ts

```diff
@@ -39,6 +41,7 @@ export default class ResultsCalculator {
 		let team2Points: number;
 
 		if (score1 === score2) {
+		
```

Comment:
The scores (i.e. 0, 1, 3) should be constants, declared either as static values within this class, as an exportable constants object from this module, or in a config file. 

This does a couple things:

1) It allows for unit tests to use the constants, instead of hardcoding those values in the tests themselves.
2) It puts all values in one place, so the values can be (a) modified in one place only and (b) there is no chance that someone mistakenly changes the value in one place and not the other.

Reply:
On another note, it would be neat to think of a solution that works w. n teams versus just 2. That is, iterate through and sort the scores of individual games / matches for n teams, and then using a scoring matrix, add the appropriate point values to each team. This code could be refactored to accomodate for such a scenario.